models/Issue.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+var mongoose = require('mongoose');
+
+var issueSchema = mongoose.schema({
+	author_id: Number,
+	title: String,
+	content: String,
+	votes: {
+		up: Number,
+		down: Number
+	},
+	date_created: String
```

Comment:
Also on date, we should make it a Date object and then parse it to the format Randy wants when we send it to him. I don't know if Mongoose can parse a date string when sorting.